src/ofp_arp.c

```diff
@@ -767,7 +767,7 @@ int ofp_arp_init_tables(void)
 		odp_rwlock_write_lock(&shm->arp.set[i].table_rwlock);
 	odp_rwlock_write_lock(&shm->arp.fr_ent_rwlock);
 
-	for (i = 0; i < NUM_ARPS; ++i)
+	for (i = 1; i < NUM_ARPS; ++i)
```

Comment:
Perhaps it would be good to add a comment here to state why the first index is skipped.

Reply:
Ok... I'll add a comment.

Reply:
While you do that you can also add the reviewed-by tags, so this PR can be merged.